pkgs/servers/firebird/2_5.nix

```diff
@@ -0,0 +1,48 @@
+{ autoreconfHook
+, fetchFromGitHub
+, firebird
+, icu
+, lib
+, libedit
+, stdenv
+
+, superServer ? false
+}:
+
+stdenv.mkDerivation rec {
+  pname = "firebird";
+  version = "2.5.9";
+
+  src = fetchFromGitHub {
+    owner = "FirebirdSQL";
+    repo = "firebird";
+    rev = "R${builtins.replaceStrings [ "." ] [ "_" ] version}";
```

Comment:
```suggestion
    rev = "R${lib.replaceStrings [ "." ] [ "_" ] version}";
```

Reply:
Why is this relevant? Should `builtins` not be used?

Reply:
If there is an equivalent function under lib that one should be used because it may have some backwards compatibility code.

Reply:
Done